File: C/retangulo.c
#include <stdio.h>
#include <math.h>

int main()
{
    double b_ret, a_ret, area, perimetro, diagonal;

    printf("digite a base do retangulo:");
    scanf("%lf", &b_ret);

    printf("digite a altura do retangulo:");
    scanf("%lf", &a_ret);

    area = b_ret*a_ret;
    perimetro = 2*b_ret+2*a_ret;
    diagonal = sqrt(b_ret*b_ret + a_ret*a_ret);

    printf("area: %.4lf\n", area);
    printf("perimetro: %.4lf\n", perimetro);
    printf("diagonal: %.4lf\n", diagonal);
    
    return 0;
}

File: Java/retangulo.java
import java.util.Scanner;
import java.util.Locale;

public class retangulo {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Locale.setDefault(Locale.US);

        double base, altura, area, perimetro, diagonal;

        System.out.print("digite a base do retangulo: ");
        base = sc.nextDouble();

        System.out.print("digite a altura do retangulo: ");
        altura = sc.nextDouble();

        area = base*altura;
        perimetro = 2*base * 2*altura;
        diagonal = Math.sqrt(base*base + altura*altura);

        System.out.println("base: "+base+
        "\naltura: "+altura+
        "\narea: "+String.format("%.4f", area)+
        "\nperimetro: "+String.format("%.4f", perimetro)+
        "\ndiagonal: "+String.format("%.4f", diagonal) );
        
        sc.close();
    }
}
